docs/docs/cmd/tenant/report/report-settings-set.mdx

```diff
@@ -0,0 +1,69 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# report settings set
+
+Update (adminReportSettings) tenant-level settings for Microsoft 365 reports
+
+## Usage
+
+```sh
+m365 tenant report settings set [options]
+```
+
+## Options
+
+```md definition-list
+`-h, --hideUserInformation  <hideUserInformation>`
+: Determines whether all reports conceal user information such as usernames, groups, and sites.  
+
+`--verbose`
```

Comment:
You can remove --verbose and --debug. They are part of the `<Global />` component